### DISCUSSION TITLE
How to implement stripe subscription payment ?

### DISCUSSION THREAD

**Discussion by Satishpethani92** [flutter-stripe/flutter_stripe#625] [Q&A]
I would like to implement subscription payment using this SDK so it is possible ?

**Comment by remonh87** [comment]
yes this is possible. I recommend you to follow the official docs: https://stripe.com/docs/billing/subscriptions/overview#integration-example

**Reply by Satishpethani92** [reply]
I know this is possible in the Web but i am asking about the mobile SDK.

**Reply by fldvlpr** [reply]
@Satishpethani92 Were you able to create subscription from mobile flutter side using stripe sdk in flutter?

**Reply by Satishpethani92** [reply]
yes

**Reply by Satishpethani92** [reply]
You can get stripe subscription using the api and then use in the payment intent.

**Reply by fldvlpr** [reply]
@Satishpethani92 Can you provide the list of steps one by one using flutter stripe package including backend api calls?

**Reply by dotpngn** [reply]
@fldvlpr i made this sample project using stripe, hope it help: https://github.com/gbrlparada/flutter-stripe-subscription

**Comment by joelsongsouzza** [comment]
I still didn't get it how to use with the package. I managed to use the payment intent, but I don't know how to charge monthly as subscription.

**Reply by awatson32** [reply]
I am also interested in this.

**Reply by fldvlpr** [reply]
@joelsongsouzza @awatson32 Were you able to create subscription from mobile flutter side using stripe sdk in flutter?

**Comment by gurulnd-git** [comment]
I am also struggling to implement subscription from mobile SDK with cloud function... I was happy with previous stripr_payments plugin where we can get a response on payment success or creating subscription.... now everything returns void.... with out payment sheet how to make a payment for subscription?

**Reply by fldvlpr** [reply]
@gurulnd-git Were you able to create subscription from mobile flutter side using stripe sdk in flutter?

**Comment by evansmwenda** [comment]
I have found this to be helpful abit... You can check it out

https://mypace-co-ltd.medium.com/use-flutter-stripe-for-payment-and-subscription-registration-without-a-backend-9c503ed86fdd

**Reply by hacktolove** [reply]
This is not working due to the following stripe error :
`
{error: {message: Sending credit card numbers directly to the Stripe API is generally unsafe. We suggest you use test tokens that map to the test card you are using, see https://stripe.com/docs/testing. To enable raw card data APIs in test mode, see https://support.stripe.com/questions/enabling-access-to-raw-card-data-apis., request_log_url: https://dashboard.stripe.com/test/logs/req_Mt3CtmtTuKIoFB?t=1715638500, type: invalid_request_error}}
`

**Comment by dotpngn** [comment]
I made a simple example of how you can create a subscription, you can take a look here: https://github.com/gbrlparada/flutter-stripe-subscription

**Comment by Satishpethani92** [comment]
There is any trick for app approve with stripe payment integration in apple store ?
